Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -200,33 +196,35 @@ private void RegisterFactories(IWindsorContainer container, Assembly[] assemblie
         {
             foreach (var assembly in assembliesToRegister)
             {
-                var assemblyTypes = GetIoCRegisteredTypes(assembly);
-
-                container.Register(Types.From(assemblyTypes)
-                    .Where(type => type.IsInterface && type.Name.EndsWith("Factory"))
+                container.Register(Types.FromAssembly(assembly)
+                    .IncludeNonPublicTypes()
+                    .Where(type => type.IsInterface && type.Name.EndsWith("Factory") && type.NotDisabledExperimental(_initialSettings))
+                    .WithService.Self()
                     .Configure(c => c.AsFactory())
                     .LifestyleSingleton());
             }
         }
 
         private static void RegisterSpecialFactories(IWindsorContainer container)
         {
-            container.Register(Component.For<IFolderBrowserFactory>()
-                .ImplementedBy<DialogFactory>()
+            container.Register(Component.For<IDeclarationFinderFactory>()
+                .ImplementedBy<DeclarationFinderFactory>()
```

Comment:
Why does that get bound to a concrete implementation? This concrete implementation only exists for the parser mock.
If at all, please use the concrete implementation corresponding to the registration of `DeclarationFinder`, which is `ConcurrentlyConstructedDeclarationFinderFactory`.